PlayList.java

```diff
@@ -35,32 +35,53 @@ public Track getTrack(int index) {
      *  If the list is full, does nothing and returns false.
      *  Otherwise, appends the track and returns true. */
     public boolean add(Track track) {
-        //// replace the following statement with your code
+        if(this.size>=this.maxSize)
+            return false;
+        this.tracks[size]=track;
+        this.size++;
         return true;
     }
 
     /** Returns the data of this list, as a string. Each track appears in a separate line. */
     //// For an efficient implementation, use StringBuilder.
-    public String toString() {
-        //// replace the following statement with your code
-        return "";
+    public String toString() 
+    {
+        if(size==0)
+            return"";
+        String str = this.tracks[0].toString();
+        for (int i=1;i<this.size ;i++ ) 
+        {
+            str+= "/n" + this.tracks[i].toString();
+        }
+        return str;
     }
 
     /** Removes the last track from this list. If the list is empty, does nothing. */
-     public void removeLast() {
-        //// replace this comment with your code
-    }
+     public void removeLast() 
+     {
+        if(this.tracks[size-1]!=null&&size>0)
+        {
+            this.tracks[size-1]=null;
+            size--;  
+        }
+     }
     
     /** Returns the total duration (in seconds) of all the tracks in this list.*/
-    public int totalDuration() {
-        //// replace the following statement with your code
-        return 0;
+    public int totalDuration() 
+    {   
+        int sumedura = 0;
```

Comment:
Use meaningful names for variables and use camelCase
```suggestion
        int sumDurations = 0;
```